### PR TITLE
Import email address data into outlook 2011 for mac

### PR DESCRIPTION
**Pull request by WilliamLoy**

### CODE REVIEW

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-259da20b61/how-to/). :bow:

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-cde05eaa0b/how-to/).

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-6e8894490c/how-to/).

File: content/rackspace-email/import-email-address-data-into-outlook-2011-for-mac.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+type: article
+created_date: '2017-09-07'
+created_by: William Loy
+last_modified_date: '2017-10-06'
+last_modified_by: Stephanie Fillmon
+product: Rackspace Email
+product_url: rackspace-email
+---
+
+This article how to import a copy of your email address data from Outlook 2011 for Mac OS.
```
[the remark this answers is not in the document]
Comment (stephamon) [reply content/rackspace-email/import-email-address-data-into-outlook-2011-for-mac.md]: Haha oops

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-548c3fddb3/how-to/). :bow:

**Pull request merged**